homeassistant/components/netatmo/climate.py

```diff
@@ -328,6 +328,35 @@ def device_state_attributes(self):
 
         return attr
 
+    def turn_off(self):
+        """Turn the entity off."""
+        if self._module_type == NA_VALVE:
+            self._data.homestatus.setroomThermpoint(
+                self._data.home_id,
+                self._room_id,
+                STATE_NETATMO_MANUAL,
+                DEFAULT_MIN_TEMP,
+            )
+        elif self.hvac_mode != STATE_OFF:
+            self._data.homestatus.setroomThermpoint(
+                self._data.home_id, self._room_id, HVAC_MODE_OFF
```

Comment:
`HVAC_MODE_OFF` is a home assistant constant. We shouldn't use that with the netatmo api. Please use a netatmo specific constant.